### ISSUE TITLE
Add an options parameter to Readability constructor

### ISSUE THREAD

**Issue by n1k0** [mozilla/readability#128]
Ref https://bugzilla.mozilla.org/show_bug.cgi?id=1152219#c1

In order to ease configuration of a created Readability instance, we should add a way to define options, eg. by adding a third `options` argument to its constructor:

```js
new Readability(uri, document, {
  debug: true,
  maxElemsToParse: 5000,
  // etc.
});
```

**Comment by n1k0** [comment]
I've realized that `MAX_ELEMS_TO_PARSE` is used within chrome only, not Readability. Though that shouldn't prevent us to make Readability configurable trough constructor, methinks.

**Comment by n1k0** [comment]
Especially as preventing the parser to parse too large documents according to some passed option sounds like a good idea, as we would be able to add unit tests for this much more easily. I'll work on that.

**Comment by n1k0** [comment]
Fixed by #128 (4424b0bad75871eb63b0693e40af93ec0dbf2c17)
